frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout.ts

```diff
@@ -0,0 +1,38 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
```

Comment:
```suggestion
import 'Frontend/demo/init'; // hidden-source-line
```
Change this comment wherever it appears in every example.
https://github.com/vaadin/docs-app/pull/256

Reply:
Updated.